src/components/Person/Person.jsx

```diff
@@ -1 +1,15 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">{`My name is ${person.name}`}</h2>
+    {person.age && (
+      <p className="Person__age">{`I am ${person.age}`}</p>
+    )}
+    <p className="Person__partner">
```

Comment:
Would be great to add spaces between multiline siblings for a better readability 
```suggestion
    )}
    
    <p className="Person__partner">
```